x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/use_columns.tsx

```diff
@@ -209,19 +213,24 @@ const INTEGRATIONS_COLUMN: TableColumn = {
   truncateText: true,
 };
 
-const useActionsColumn = (): EuiTableActionsColumnType<Rule> => {
-  const actions = useRulesTableActions();
+const useActionsColumn = ({
+  showExceptionsDuplicateConfirmation,
+}: ActionColumnsProps): EuiTableActionsColumnType<Rule> => {
+  const actions = useRulesTableActions({ showExceptionsDuplicateConfirmation });
```

Comment:
Is it possible to call the `useBulkDuplicateExceptionsConfirmation` hook within `useActionsColumn` or `useRulesTableActions`? That way you can avoid having to modify the props for `useMonitoringColumns` and `useRulesColumns`

Reply:
Something like:
```
const useActionsColumn = (): EuiTableActionsColumnType<Rule> => {
  const { showBulkDuplicateConfirmation } = useBulkDuplicateExceptionsConfirmation();
  const actions = useRulesTableActions({
    showExceptionsDuplicateConfirmation: showBulkDuplicateConfirmation,
  });
```

Reply:
It's probably not possible currently if we do not rewrite this hook to use a shared state between components. (like context or something)

It can make things a bit more complex then, do you think we need to go with this direction?